jgt_tools/env_setup.py

```diff
@@ -20,6 +20,9 @@ def env_setup(verbose):
     if os.getenv("VIRTUAL_ENV"):
         print(f"Setting up Virtual Environment: {os.environ['VIRTUAL_ENV']}")
     print()
+    for command in ("build_docs", "run_tests", "env_setup"):
+        if DEFAULT_CONFIGS[f"{command}_commands"] == CONFIGS[f"{command}_commands"]:
+            __commands_to_run.insert(2, f"poetry run pip install jgt_tools[{command}]")
```

Comment:
This code is puzzling me.  Due to `<reasons>` I haven't been able to pull it and play with it,
but if I follow the logic, only if the commands to run for any of the three "extras" on line 23 exactly matches the default values, will they be installed? I would have expected to see a check for the `{command}_commands` key to exist in the CONFIGs object for the basis of installation.  Again, hypothetically since I haven't pulled it to play around, but if someone using jgt_tools has different `run_tests` commands, they won't get `run_tests` at all?
That is, if they change the command, even just to pass in other flags or change the verbosity level?

RR to change my comment once I can pull and play with this, but that might be another day, so wanted to leave brain droppings for potential discussion.

Reply:
I've added a comment block that will hopefully better explain, both now and in the future, the intent of this code.